app/pages/search/Sort.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { injectT } from 'i18n';
+import SelectControl from './controls/SelectControl';
+
+class Sort extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      selected: '',
+    };
+
+    const { lang } = props;
+    this.sortOptions = [
+      { label: 'Name', value: `resource_name_${lang}` },
+      { label: 'Type', value: 'type' },
+      { label: 'Premises', value: `unit_name_${lang}` },
+      { label: 'People', value: 'people_capacity' },
+      { label: 'Open now', value: 'sortByOpenNow' },
+    ];
+  }
+
+  handleChange = ({ value }) => {
+    const filters = {};
+    if (value === 'sortByOpenNow') {
```

Comment:
Use constant defined above for `sortByOpenNow `

Reply:
Corrected in new commits